ras_party/views/respondent_view.py

```diff
@@ -98,3 +98,27 @@ def change_respondent_details(respondent_id):
     payload = request.get_json()
     respondent_controller.change_respondent_details(payload, respondent_id)
     return make_response('Successfully updated respondent details', 200)
+
+
+@respondent_view.route('/respondents/claim')
+def validate_respondent_claim():
+    """Validates if the respondent has a claim on a specific business and survey
+    Both valid and invalid claims return 200 with the difference being indicated in the response body.
+
+    Not RESTFUL style more like RPC, but the alternative is for duplication of business logic
+    in client services. There is an argument to use a 403 on invalid claims , but that should be a status on the
```

Comment:
Yeah, as discussed on the other PR the status codes are somewhat left wanting in this situation!